tt_eager/tt_dnn/op_library/prod/kernels/dataflow/writer_prod_nc.cpp

```diff
@@ -0,0 +1,31 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
This seems like duplicate of `tt_eager/tt_dnn/kernels/dataflow/writer_unary_interleaved_start_id.cpp`.
Can we not use this instead? Or do you expect them to differ in the future?

Reply:
Updated the file